app/controllers/medicaid/insurance_current_type_controller.rb

```diff
@@ -1,15 +1,80 @@
 # frozen_string_literal: true
 
 module Medicaid
-  class InsuranceCurrentTypeController < Medicaid::ManyMemberStepsController
+  class InsuranceCurrentTypeController < MedicaidStepsController
+    helper_method :current_member
+
+    def update
+      @step = step_class.new(step_params)
+
+      if @step.valid?
+        current_member.update(step_params)
+        redirect_to(next_path)
+      else
+        render :edit
+      end
+    end
+
+    def current_member
```

Comment:
feels like there's a lot happening in this method that should have specs -- maybe controller specs?

Reply:
There were controller specs: https://github.com/codeforamerica/michigan-benefits/pull/417/files#diff-01f7d54e987c44ab7c743f3b92c48b1bR4